caf-audit/src/main/java/com/hpe/caf/auditing/AuditEventBuilder.java

```diff
@@ -38,6 +38,13 @@ void addEventParameter(
         String value
     );
 
+    void addEventParameter(
+            String name,
+            String columnName,
+            String value,
+            AuditIndexingHint indexingHint
+    ) throws Exception;
```

Comment:
Let's provide a default implementation which ignores the indexing hint, so that the interface change is not a breaking change.